data/gallery.js

```diff
@@ -37,6 +37,40 @@ myAlbum.appendChild(description);
 
 document.body.appendChild(myAlbum);
 
+const imagesUl = document.createElement('ul');
+console.log(imagesUl);
 
+// function imagesLi(album){
+//     console.log(album);  
+//     document.createElement('li');  
+// }
+//  album.images.forEach(imagesLi);
+
+// const myImages =document.createElement('li');
+// console.log(myImages);
+
+// album.images.forEach(function(myAlbums) {
+//     document.createElement('li');
+//     console.log(myAlbums);
+// }
+// my)
+function logImages(images){
+    const imagesLi = document.createElement('li');
+      imagesLi.textContent = images.title;
+    const imagesImg = document.createElement('img');
+      imagesImg.src = images.src;
```

Comment:
You should get into the habit of adding an 'alt' tag to images (it's required for accessibility), so here you could use the image title as the alt tag.

Reply:
I did think this. I will add them in.